documentation/docs/98-reference/54-types.md

```diff
@@ -154,4 +154,73 @@ The `app.d.ts` file is home to the ambient types of your apps, i.e. types that a
 
 Always part of this file is the `App` namespace. This namespace contains several types that influence the shape of certain SvelteKit features you interact with.
 
-> TYPES: App
+## Error
```

Comment:
This looks like an accidental change that should be reverted

Reply:
Changes reverted!